README.md

```diff
@@ -1,71 +1,50 @@
-# Nuked SC-55 CLAP audio plugin
+# Nuked SC-55 CLAP
 
-The plugin is built upon [J.C. Moyer's fork](https://github.com/jcmoyer/Nuked-SC55)
-of nukeykt's original [Nuked SC-55](https://github.com/nukeykt/Nuked-SC55)
-project.
-
-The plugin aims to preserve an important part of DOS gaming history for all to
-freely enjoy for posterity. It's only intended for **personal use** (e.g.,
-retro gaming or writing music as a hobby) and **research purposes**.
-
-As per the original Nuked SC-55 license, neither the code nor the published
-binaries may be used directly or indirectly for the creation of commercial
-Roland SC-55 emulation hardware boxes. Moreover, any use of the software in
-commercial music production is prohibited, and so is including the plugin in
-any commercial software package.
+Nuked SC-55 for the CLever Audio Plug-in ([CLAP](https://cleveraudio.org/)) is built upon [J.C. Moyer's fork](https://github.com/jcmoyer/Nuked-SC55) of nukeykt's original [Nuked SC-55](https://github.com/nukeykt/Nuked-SC55) project.
```

Comment:
Let's do "The Nuked SC-55 for ([CLAP](https://cleveraudio.org/)) audio plugin...". The rest is fine 👍🏻